openquake/hazardlib/geo/surface/base.py

```diff
@@ -227,42 +227,35 @@ def get_rx_distance(self, mesh):
             Numpy array of distances in km.
         """
         top_edge = self.mesh[0:1]
-        xxx = _get_finite_top_rupture(mesh)
```

Comment:
Why are you removing this one?

Reply:
`flake8` told me that it was unused, now I sent that you meant to use it.